roles/grafana/tasks/main.yaml

```
@@ -60,3 +60,33 @@
     mode: '0640'
   notify: _tina_restart_grafana
   when: grafana__datasources_loki
+
+- name: Create dashboard provider
+  ansible.builtin.template:
+    src: dashboards.yaml.j2
+    dest: '{{ grafana__paths__provisioning }}/dashboards/dashboards.yaml'
+    owner: root
+    group: grafana
+    mode: '0640'
+  notify: _tina_restart_grafana
+  when: grafana__dashboards
+
+- name: Create dashboards directory
+  ansible.builtin.file:
+    path: '{{ grafana__dashboards_path }}'
+    state: directory
+    owner: grafana
+    group: grafana
+    mode: '0750'
```

same here; plus, grafana does not need to be able to write
```suggestion
    owner: root
    group: root
    mode: '0755'
```